Rest/Field/Value.php

```diff
@@ -38,28 +35,31 @@ class Value
     /** @var \Psr\Log\LoggerInterface */
     protected $logger;
 
+    /** @var array */
+    protected $parameters;
```

Comment:
It was like that before, and the use-case was clear, why would you change it ? The idea was that extra parameters can be added without altering the service's signature.

We usually don't ask in pull request for improvements to code that already exists unless it actually improves something.